examples/pipeline-example.yaml

```
@@ -42,6 +42,33 @@
       - [batch2.input.training-labels*, train.input.training-labels]
       - [batch2.metadata.optimal_learning_rate, train.parameter.learning_rate]
       - [batch2.output.processed-images*, train.input.training-images]
+- pipeline:
+    name: My deployment pipeline
+    nodes:
+      - name: batch1
+        type: execution
+        step: Batch feature extraction
+      - name: train
+        type: execution
+        step: Train model
+      - name: deploy
+        type: deployment
+        deployment:
+          name: MyDeployment
```

Considering all the below requests, could a simpler shape for this be

```suggestion
      - name: deploy
        type: deployment
        deployment: MyDeployment
        endpoints:
          - predict-digit
```
or am I missing something?